src/NzbDrone.Core/ImportLists/Simkl/SimklAPI.cs

```diff
@@ -23,11 +24,17 @@ public class SimklSeriesPropsResource
     public class SimklSeriesResource
     {
         public SimklSeriesPropsResource Show { get; set; }
+
+        [JsonProperty(PropertyName = "anime_type")]
```

Comment:
For consistency with the other uses in this file you can change to `[JsonProperty("anime_type")]`